components/learn/TheCarefulExplanationForBeginners.vue

```diff
@@ -1,12 +1,7 @@
 <template>
   <CarefulExplanation v-bind="$attrs">
     <template #summary>
-      <h2 class="copy__subtitle">
-        A Carefully Worded Summary of Quantum Computing for Beginners
-      </h2>
-      <p class="copy__paragraph">
-        Physics is an attempt to describe the behaviour of the world around us. Around the early 20th century, through delicate experiments in laboratories, physicists saw strange behaviours which could not be explained by their current understanding of physics. These discoveries led to the development of the more complete ‘quantum’ physics, which was able to describe this behaviour very successfully.
-      </p>
+      <AppDescriptionCard :title="title" :description="preliminarDescription" />
```

Comment:
should this be `preliminaryDescription` or `primaryDescription`?

Reply:
I thought it is 'preliminary' because it is the 'introductory paragraph'. With 'primary' I understand that there are secondary paragraphs (more o less like h1, h2, ...). But you're the native English, if you say 'primary' is better, I trust you :). Should I change it?

Reply:
Good point, and I'm okay with it... if we can just update it to `preliminaryDescription` (just needs the `y`)

Reply:
It's true!! thank you!! good catch!